pages/common/docker-compose.md

```diff
@@ -0,0 +1,27 @@
+# docker-compose
+
+> Run and manage multi container docker applications
+
+- Create and start containers in the background using a local `docker-compose.yml` file:
+
+`docker-compose up -d`
+
+- Start containers, rebuild if necessary:
+
+`docker-compose up --build`
+
+- Start containers using a custom configuration file:
+
+`docker-compose --file {{path/to/config}} up`
+
+- Stop running containers:
+
+`docker-compose stop`
+
+- Stop and remove containers, networks, images, and volumes:
```

Comment:
What's the difference between this command and the previous one? Does it delete containers / networks / images / volumes?

Reply:
@sbrl Yes, the `down` command stops and deletes the containers. The word `remove` is more common in docker (i.e `docker rm`) so IMO its better then `delete`.

Reply:
Ah, ok - sounds good to me :smiley_cat: